app/src/main/java/com/kickstarter/ui/activities/ChangeEmailActivity.kt

```diff
@@ -58,6 +59,28 @@ class ChangeEmailActivity : BaseActivity<ChangeEmailViewModel.ViewModel>() {
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe { showErrorSnackbar(change_email_layout, it) }
 
+        this.viewModel.outputs.isCreator()
+                .compose(bindToLifecycle())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe {
+                    verification_text_view.text = getString(R.string.Send_verfication_email)
```

Comment:
what if they're already verified? we want to show this row? when does it ever hide? 🤔